checks.d/openstack.py

```diff
@@ -319,6 +343,11 @@ def get_stats_for_single_hypervisor(self, hyp_id):
             'hypervisor_id:{0}'.format(hyp['id']),
             'virt_type:{0}'.format(hyp['hypervisor_type'])
         ]
+        if hyp['hypervisor_hostname'] in self._aggregate_list:
+            service_check_tags.append('aggregate:{0}'.format(self._aggregate_list[hyp['hypervisor_hostname']]['aggregate']))
+            # Need to check if there is a value for availability_zone because it is possible to have an aggregate without an AZ
+            if self._aggregate_list[hyp['hypervisor_hostname']]['availability_zone']:
```

Comment:
The key will exist but it will either be an empty string or None. I didn't want to send the empty tag so I put this conditional around it.